mkosi

```diff
@@ -3051,7 +3051,7 @@ def parse_args() -> CommandLineArguments:
     group.add_argument("--password", help='Set the root password')
 
     group = parser.add_argument_group("Host configuration")
-    group.add_argument("--extra-search-paths", action=ColonDelimitedListAction, default=[],
+    group.add_argument("--extra-search-path", dest='extra_search_paths', action=ColonDelimitedListAction, default=[],
```

Comment:
the internal field should probably also drop the "s"

do we care about compat here btw? if this has been added more recently than the last release I figure we don't need to care

Reply:
@cmarcelo do you think we need the compat option here?

@poettering I don't think the way you did for `--kernel-commandline` works... that creates a `dest.kernel_commandline` that we don't use anywhere. Gotta check if we can have multiple options setting the same dest.